schemas/core/components/terms.json

```diff
@@ -46,6 +46,14 @@
       "type": "boolean",
       "description": "A flag indicating whether this booking will be reconciled once the trip has finished"
     },
+    "cancellable": {
```

Comment:
No this needs to be an object, that supports fee and other possible condition

Reply:
@blackevil245  Added as object, not aware of any other cancellation related conditions that fee, you have anything in mind?

Reply:
Hmm not at the moment that i know of. But i reckon it would be available for other TSPs